core/src/main/java/org/elasticsearch/rest/action/cat/RestSnapshotAction.java

```diff
@@ -52,9 +52,12 @@ public RestSnapshotAction(Settings settings, RestController controller, Client c
 
     @Override
     protected void doRequest(final RestRequest request, RestChannel channel, Client client) {
-        GetSnapshotsRequest getSnapshotsRequest = new GetSnapshotsRequest();
-        getSnapshotsRequest.repository(request.param("repository"));
-        getSnapshotsRequest.snapshots(new String[] { GetSnapshotsRequest.ALL_SNAPSHOTS });
+        GetSnapshotsRequest getSnapshotsRequest = new GetSnapshotsRequest()
+                .repository(request.param("repository"))
+                .snapshots(new String[]{GetSnapshotsRequest.ALL_SNAPSHOTS});
+
+        getSnapshotsRequest.ignoreUnavailable(request.paramAsBoolean("ignore_unavailable", getSnapshotsRequest.ignoreUnavailable()));
```

Comment:
This new parameter should be documented in API for /_cat/snapshots (file cat.snapshots.json)